packages/dicomImageLoader/src/imageLoader/wadouri/loadImage.ts

```diff
@@ -23,6 +23,19 @@ function addDecache(imageLoadObject: Types.IImageLoadObject, imageId: string) {
   };
 }
 
+/**
+ *
```

Comment:
Perhaps add a one line description? "Loads an image from the given data set promise."